src/account/logoutActions.js

```diff
@@ -1,9 +1,15 @@
 /* @flow strict-local */
 import * as NavigationService from '../nav/NavigationService';
-import type { AllAccountsAction, ThunkAction } from '../types';
-import { LOGOUT } from '../actionConstants';
+import type { PerAccountAction, AllAccountsAction, ThunkAction } from '../types';
+import { LOGOUT, RESET_ACCOUNT_DATA } from '../actionConstants';
 import { resetToAccountPicker } from '../nav/navActions';
 
+// Redefined here, instead of imported from accountActions, just to prevent
+// an import cycle.
+const resetAccountDataPlain = (): PerAccountAction => ({
+  type: RESET_ACCOUNT_DATA,
+});
```

Comment:
Hmm, that's a bit awkward.

It's convenient that we generally have just one action creator (outside tests) for each action type, because it makes it straightforward to find all the places we create such actions by finding that creator's call sites. So that'd be nice to preserve here.

It looks like the cycle can be fixed by moving the single definition here into `logoutActions.js` (so having `accountActions.js` import it from here.) That's inspired by this comment we have on the only existing export in this file:
```js
// In its own file just to prevent import cycles.
export const logout = (): ThunkAction<Promise<void>> => async (dispatch, getState) => {
```

I think "reset account data" is thematically close enough to "log out" that this file is a fine home for it.